src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -80,14 +80,14 @@ def create_resource_group(rg_name, location, tags=None):
     return rcf.resource_groups.create_or_update(rg_name, parameters)
 
 
-def create_appliance(resource_group_name,
-                     appliance_name, managedby_resource_group_id,
-                     location, kind, managedapp_definition_id=None,
-                     plan_name=None, plan_publisher=None, plan_product=None,
-                     plan_version=None, tags=None, parameters=None):
+def create_application(resource_group_name,
+                       application_name, managedby_resource_group_id,
+                       location, kind, managedapp_definition_id=None,
```

Comment:
`location` should be optional with the defaults using resource group's

Reply:
Done